release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1220,6 +1220,8 @@ In the following tables, features are marked with the following statuses:
 
 * {run-once-operator} (RODOO) cannot be installed on clusters managed by the Hypershift Operator. (link:https://issues.redhat.com/browse/OCPBUGS-17533[*OCPBUGS-17533*])
 
+* In hosted control planes for {product-title}, if you disable the ingress capability, the Console Operator reports the following error message: `RouteHealthAvailable: failed to GET route`. To avoid this error, do not disable the ingress capability in an {product-title} managed cluster. (link:https://issues.redhat.com/browse/OCPBUGS-33788[*OCPBUGS-33788*])
```

Comment:
```suggestion
* In hosted control planes for {product-title}, if you disable the ingress capability, the Console Operator returns the following error message: 
+
[source,text]
----
RouteHealthAvailable: failed to GET route. 
----
+
To avoid this error, do not disable the ingress capability in an {product-title} managed cluster. (link:https://issues.redhat.com/browse/OCPBUGS-33788[*OCPBUGS-33788*])
```

Reply:
I like this suggestion. Addressed!